src/main/resources/org/jenkinsci/plugins/pipeline/utility/steps/json/ReadJSONStep/help.html

```diff
@@ -0,0 +1,36 @@
+<!--
+  ~ The MIT License (MIT)
+  ~
+  ~ Copyright (c) 2016 Nikolas Falco
+  ~
+  ~ Permission is hereby granted, free of charge, to any person obtaining a copy
+  ~ of this software and associated documentation files (the "Software"), to deal
+  ~ in the Software without restriction, including without limitation the rights
+  ~ to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+  ~ copies of the Software, and to permit persons to whom the Software is
+  ~ furnished to do so, subject to the following conditions:
+  ~
+  ~ The above copyright notice and this permission notice shall be included in all
+  ~ copies or substantial portions of the Software.
+  ~
+  ~ THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+  ~ IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+  ~ FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+  ~ AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+  ~ LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+  ~ OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+  ~ SOFTWARE.
+  -->
+<p>
+    Reads a file in the current working directory or a String as a plain text
+    <a href="http://www.json.org/json-it.html" target="_blank">JSON</a> file.
+    The returned object is a normal Map with String keys or a List of primitives or Map.
+</p>
+<p>
+    <strong>Example:</strong><br/>
+    <code><pre>
+def props = readJSON file: 'dir/input.json', text: '{ other: Override }'
```

Comment:
In `help-text.html` and `help-file.html` you state that `file` or `text` can be used, not both at the same time. Yet this example uses both.
Also I don't think the text will parse since `Override` isn't surrounded by quotes.

The use of Override is also a bit confusing, since the other steps in this plugin that uses it to exeplify how to **override** data in the file. So it's better to use some other string to not further confuse the user.